yt_dlp/extractor/nebula.py

```diff
@@ -478,3 +478,36 @@ def _real_extract(self, url):
             playlist_id=collection_slug,
             playlist_title=channel.get('title'),
             playlist_description=channel.get('description'))
+
+
+class NebulaSeasonIE(NebulaBaseIE):
+    IE_NAME = 'nebula:seasons'
+    _VALID_URL = rf'{_BASE_URL_RE}/(?P<sn>\w+)/?season/(?P<sid>\d+)'
+    _TESTS = [{
+        'url': 'https://nebula.tv/jetlag/season/15',
+        'info_dict': {
+            'id': 'jetlag_15',
+        },
+        'playlist_count': 6,
+    }, {
+        'url': 'https://nebula.tv/jetlag/season/14',
+        'info_dict': {
+            'id': 'jetlag_14',
+        },
+        'playlist_mincount': 6,
+    }, {
+        'url': 'https://nebula.tv/jetlag/season/10',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        season_name, season_id = self._match_valid_url(url).group('sn', 'sid')
+        video_id = f'{season_name}_{season_id}'
```

Comment:
Don't use `video_id` as the name: this is a playlist, not a video
```suggestion
        playlist_id = f'{series}_{season_number}'
```